### PR TITLE
280 website registry link does not take thyra running port into account

### PR DESCRIPTION
**Pull request by thomas-senechal**

### CODE REVIEW

**Review by peterjah — APPROVED** [review]

**Comment by peterjah** [comment]
we will have to change again theses links for the support of https ! to be continued !

**Comment by thomas-senechal** [comment]
> we will have to change again theses links for the support of https ! to be continued !

I'm not sure about that. We can remove the "http:" part and the web browser will figure out by itself which protocol it should use. By default, it will try to use `https`. 

I just tried it and it's working fine. I will push that fix.

File: int/api/html/front/registry.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ function tableInsert(website) {
         .getElementById("website-deployers-table")
         .getElementsByTagName("tbody")[0];
     const row = tBody.insertRow(-1);
-    const url = "http://" + website[NAME] + ".massa/";
+    const url = `//${website[NAME]}.massa:${window.location.port}/`;
```
Comment (gregLibert) [inline int/api/html/front/registry.js:27]:
```suggestion
    const port = window.location.port == 80 ? "" : ":" + window.location.port;
    const url = `${windows.location.protocol}//${website[NAME]}.massa${port}/`;
```

File: int/api/html/front/website.js [hunk 2]
```diff
@@ -162,7 +162,7 @@ async function tableInsert(resp, count) {
         .getElementById("website-deployers-table")
         .getElementsByTagName("tbody")[0];
     const row = tBody.insertRow(-1);
-    const url = "http://" + resp.name + ".massa/";
+    const url = `//${resp.name}.massa:${window.location.port}/`;
```
Comment (gregLibert) [inline int/api/html/front/website.js:165]: idem

**Review by gregLibert — CHANGES_REQUESTED** [review]
We will have https really soon :-)

File: int/api/html/front/registry.js (hunk 1 above)
Comment (thomas-senechal) [reply int/api/html/front/registry.js:27]: I'm not sure think we need to add the `protocol` at the beginning since

**Review by gregLibert — APPROVED** [review]
:+1:

**Pull request merged**